### PR TITLE
This week in os #21

### PR DESCRIPTION
**Pull request by emmasofiah2o**
- Added file,
- Update config

### CODE REVIEW

File: src/twios/2023-03-27.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+- [mainmatter/mainmatter-website-mailer] [#20](https://github.com/mainmatter/mainmatter-website-mailer/pull/20) enable Zapier automation via email ([@marcoow])
+- [mainmatter/mainmatter-website-mailer] [#19](https://github.com/mainmatter/mainmatter-website-mailer/pull/19) make the service optional ([@marcoow])
+- [mainmatter/mainmatter-website-mailer] [#18](https://github.com/mainmatter/mainmatter-website-mailer/pull/18) send service in subject ([@marcoow])
+- [mainmatter/mainmatter-website-mailer] [#9](https://github.com/mainmatter/mainmatter-website-mailer/pull/9) include selected service in message ([@marcoow])
+- [mainmatter/renovate-config] [#14](https://github.com/mainmatter/renovate-config/pull/14) remove helpers:pinGitHubActionDigests ([@mansona])
+
+## Unknown
+
+- [dimforge/rapier.rs] [#61](https://github.com/dimforge/rapier.rs/pull/61) Update "Getting Started page" for bevy_rapier to bevy 0.10 ([@BobrImperator])
```
Comment (emmasofiah2o) [inline src/twios/2023-03-27.md]: @BobrImperator where does this go?

File: src/twios/2023-03-27.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+- [mainmatter/compare-fixture] [#3](https://github.com/mainmatter/compare-fixture/pull/3) Make sure the exitCode is 1 when diffs are detected ([@mansona])
+- [mainmatter/compare-fixture] [#2](https://github.com/mainmatter/compare-fixture/pull/2) add initial implementation ([@mansona])
```
Comment (emmasofiah2o) [inline src/twios/2023-03-27.md]: @mansona could you please help me categorize your links from this section?

File: src/twios/2023-03-27.md (hunk 1 above)
Comment (BobrImperator) [reply src/twios/2023-03-27.md]: I've a hard time deciding honestly 😅 

On one side it's Rust, but on the other the PR only changes some documentation.
There's also no point in a new label :D 

It's either ignore or `Rust` I feel like.

File: src/twios/2023-03-27.md (hunk 2 above)
Comment (mansona) [reply src/twios/2023-03-27.md]: compare-fixture is a javascript thing, so is mocha-diff

and we could probably ignore `mansona/twitch-overlay` 🤔 it's not really something for the company open source post

File: src/twios/2023-03-27.md (hunk 1 above)
Comment (emmasofiah2o) [reply src/twios/2023-03-27.md]: I'll go with rust :D

File: src/twios/2023-03-27.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+  renovatebot configuration ([@mansona])
+- [mainmatter/ember-asset-size-action]
+  [#50](https://github.com/mainmatter/ember-asset-size-action/pull/50) Update
+  all Mainmatter references ([@mansona])
+- [mainmatter/ember-asset-size-action]
+  [#49](https://github.com/mainmatter/ember-asset-size-action/pull/49) Set a
+  "minimum change" of 10 bytes to avoid jitter in reporting ([@mansona])
+- [mainmatter/ember-asset-size-action]
+  [#48](https://github.com/mainmatter/ember-asset-size-action/pull/48) Wrap the
+  file tables in a <details> ([@mansona])
```
Comment (mansona) [inline src/twios/2023-03-27.md]:
```suggestion
  file tables in a `<details>` ([@mansona])
```

**Review by mansona — APPROVED** [review]
Looks good 🎉

**Pull request merged**